meerk40t/core/svg_io.py

```diff
@@ -646,40 +670,53 @@ def parse(self, element, context_node, e_list):
         else:
             # Check if SVGElement:  Note.
             tag = element.values.get(SVG_ATTR_TAG)
+            # We need to reverse the meerk40t:... replacement that the routine had already applied:
             if tag is not None:
                 tag = tag.lower()
+                torepl = "{" + MEERK40T_NAMESPACE.lower() + "}"
+                if torepl in tag:
+                    oldval = element.values.get(tag)
+                    replacer = MEERK40T_XMLS_ID.lower() + ":"
+                    tag = tag.replace(torepl, replacer, )
+                    element.values[tag] = oldval
+                    element.values[SVG_ATTR_TAG] = tag
```

Comment:
It would probably be best to include the namespace and maybe define "note" and "operation" as constants at the top, and cover the older files that should still be *mostly* valid and wouldn't include that namespace.

Reply:
I couldn't figure out how to reverse the xmls Namespace subject, any hints are welcome (e.g. for the inkscape-label subject as well as for others)

Reply:
I never figured it out either, the "helpful" nature of that was just weirdly confusing and kind of annoying. And really the whole thing might be better off without respecting the namespace there since we're not going to run into other weird svgs with a "note" or "operation" node. Which is generally what a namespace is good for, making sure you don't clash with the same nodes for other variant files. I'm good just biting the bullet there, and calling things whatever I want without conforming to the whole namespace thing. SVG is supposed to just ignore `foreignObject` or treat them like they are whatever nodes. So long as we don't call something the same thing as a thing that's in real svg this would probably be fine.

Adhering to the the namespace stuff gains us nothing, but costs us writing backwards compatible files.

Reply:
I get that, still I think it makes sense to clearly tag mk stuff to not get into the way of any other svg writing programs.